src/main/resources/resources/logisim/fr/analyze.properties

```diff
@@ -85,20 +85,10 @@ oscillateError = Le circuit oscille.
 # model/Parser.java
 #
 implicitAndOperator = (AND Implicite)
-<<<<<<< HEAD
```

Comment:
The use of Unicode-encoded (`\udddd` notation) characters in `HEAD` was intentional as Java tools up to JDK 8 are only capable to process plain ASCII and Latin-1-encoded files. If we give up JDK 8 compatibility, we can convert all `*.properties` files to UTF-8, which makes maintaining the internationalisation files easier, as their conversion back to UTF-8 and forth to plain-ASCII using the `native2ascii` tool could be skipped.

For the moment, I would prefer if you could process all the files touched by the failed merge operation again with `native2ascii` from JDK 8.